forecast-hub-report.Rmd

```diff
@@ -76,16 +82,11 @@ pred <- load_submission_files(dates = forecast_dates,
   dplyr::filter(target_end_date >= "2020-10-17") %>%
   dplyr::select(-location)
 
-
-if (load_from_server) {
-  get_data(load_from_server = TRUE, 
-           country = "Germany_Poland")
-}
-
+get_data(load_from_server = load_from_server,
```

Comment:
if load_from_server == FALSE then this will just pointlessly load some data - hence the if-statement. It looks nicer though, so no strong preference. (would probably make sense to rework the get_data function a bit at some point).

Reply:
👍